.github/auto_assign.yaml

```
@@ -0,0 +1,4 @@
+addReviewers: true
+addAssignees: author
+reviewers:
+  mobilecoinfoundation/coredev
```

Sorry, wasn't clear, reviewers is a yaml list, so it should be:
```suggestion
  - mobilecoinfoundation/coredev
```